### PR TITLE
Fetching node identity and contract address from vault

### PR DESCRIPTION
**Pull request by suvajit-sarkar**

### CODE REVIEW

File: examples/supplychain-app/charts/expressapp-quorum/values.yaml [hunk 1]
```diff
@@ -16,7 +16,15 @@ deployment:
   #   annotations:
   #     key: "value"
   annotations: {}
-
+vault:
```
Comment (sownak) [inline examples/supplychain-app/charts/expressapp-quorum/values.yaml:19]: Add comments and example for each key here.

File: examples/supplychain-app/charts/expressapp-quorum/values.yaml [hunk 2]
```diff
@@ -34,9 +42,6 @@ expressapp:
   # The environment variables that will store the port to be working on and the end point to ask for requests
   #Eg. apiUrl: api.blockchaincloudpoc.com
   env:
-    ganache_server: http://manufacturer.test.quorum.blockchaincloudpoc.com:15021
```
Comment (sownak) [inline examples/supplychain-app/charts/expressapp-quorum/values.yaml:37]: ganache_server needs to be there, just the name should be geth_address.

File: examples/supplychain-app/quorum/express_nodeJS/Dockerfile [hunk 3]
```diff
@@ -22,4 +22,3 @@ ENV NODE_ORGANIZATION="Manufacturer"
 ENV NODE_ORGANIZATIONUNIT="manufacturer"
 
 EXPOSE ${PORT}
-CMD [ "node", "app.js" ]
```
Comment (sownak) [inline examples/supplychain-app/quorum/express_nodeJS/Dockerfile:25]: no need to change this

File: examples/supplychain-app/charts/expressapp-quorum/values.yaml (hunk 1 above)
Comment (suvajit-sarkar) [reply examples/supplychain-app/charts/expressapp-quorum/values.yaml:19]: ok will add that

File: examples/supplychain-app/charts/expressapp-quorum/values.yaml (hunk 2 above)
Comment (suvajit-sarkar) [reply examples/supplychain-app/charts/expressapp-quorum/values.yaml:37]: ok , but what should be the env var name for it ?

**Pull request merged**